Makefile

```diff
@@ -97,7 +102,7 @@ TESTIMAGE_NAME ?= $(shell $(TESTIMAGE_SCRIPT) -d)
 
 TESTIMAGE_NIX ?= $(TESTIMAGE_REGISTRY)/nix:1.2.0
 
-all: binaries crio.conf docs
+all: binaries ${BUILD_CONFIG_PATH} docs
```

Comment:
I would add `config` target, like this
```
.PHONY: config
config: ${BUILD_CONFIG_PATH}
```
and use `config` as a prerequisite here and in similar places below

Reply:
Implemented as suggested